src/backend/TrafficCourtsApi/DisputeApi.Web/Features/TicketService/TicketsService.cs

```diff
@@ -28,11 +28,12 @@ public TicketsService(ILogger<TicketsService> logger, TicketContext context)
             {
                 _context.Tickets.Add(new Ticket
                 {
-                    TicketNumber = 11234,
-                    Name = "John Doe",
-                    DateOfIssue = "11-12-2002",
-                    TimeOfIssue = "12:23",
-                    DriversLicence = "L2323G7"
+                    Id = 0002,
```

Comment:
I know this is mock and doesn't reflect what we may actually work with.  I want to understand the difference between TicketNumber, ViolcationTicketNumber and Id

Reply:
I renamed TicketNumber to ViolationTicketNumber. I found that the front end uses an Id to identify the tickets, so I included that as well.